guide/ops/upgrades/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Upgrades
+---
+
+This section provides all necessary information to upgrade Apache Brooklyn for both the RPM/DEB and Tarball packages.
+
+## Backwards Compatibility
+
+Apache Brooklyn version 0.12.0 onward runs primarily inside a Karaf container. When upgrading from 0.11.0 or below,
```

Comment:
(Not necessarily for this PR, but worth noting: we should update this wording for 1.0.0 because we'll only support the Karaf container (rather than just "primarily")).